### PR TITLE
Add empty mechanical concept - (Task #155)

### LINKED ISSUE #155: Add empty Mechanical Engineering concept

### PR DESCRIPTION
**Pull request by franzTobiasDLR**
Closes #155

### CODE REVIEW

File: de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept]: Style thingy. it should be `Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1 {` with the `{` on the same line.

File: de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1
+{
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept]: Also the concept should have the header 

`/*******************************************************************************
 * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * SPDX-License-Identifier: EPL-2.0
*******************************************************************************/`

like the other concepts.

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]

**Review by SaMuellerDLR — DISMISSED** [review]
Approved!

**Review by andriikovalov-dlr — DISMISSED** [review]

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Review by andriikovalov-dlr — APPROVED** [review]

**Pull request merged**